ts_src/psetv2/input.ts

```diff
@@ -717,6 +810,8 @@ export class PsetInput {
 
     if (!this.nonWitnessUtxo) {
       const utxo = this.witnessUtxo!;
+      if (!this.utxoRangeProof)
+        throw new Error('missing utxoRangeProof (getUtxo)');
```

Comment:
remove. In case the prevout is unconfidential, the range proof is not required at all.
I see what issue you're trying to fix, and it would be better to fix the error message raised by `confidential.unblindOutputWith*` if the conf output is missing the rangeproof